web/controllers/rules/helpers.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Dwylbot.Rules.Helpers do
+  @moduledoc """
+  Helper function
+  wait: define the time for our rules to wait depeding on the environment
+  """
+  @doc """
+    iex>waith(:prod, 1, 2, 3)
+    1
+    iex>waith(:dev, 1, 2, 3)
+    2
+    iex>waith(:test, 1, 2, 3)
+    3
+  """
```

Comment:
Nice, I think you've got a typo `waith -> wait`. You should also run this doctest in the test file for this